flaskApi/app.py

```diff
@@ -0,0 +1,77 @@
+import logging
+from zipfile import BadZipFile
+from flask import Flask, request, jsonify
+import app_service
+
+app = Flask(__name__)
+
+# Configure the logger
+logging.basicConfig(level=logging.DEBUG)
+
+
+@app.route('/upload', methods=['POST'])
+def upload_file():
```

Comment:
Separate code to small functions.  Read more: https://softwareengineering.stackexchange.com/questions/308108/when-is-it-appropriate-to-make-a-separate-function-when-there-will-only-ever-be